_dev/apps/ui/src/components/merchant-center-account/merchant-center-account-card.vue

```diff
@@ -713,6 +714,11 @@ export default {
         && this.selectedMcaDetails.name !== null
         && this.selectedMcaDetails.name !== undefined;
     },
+    hasMcaCreatedFromModule() {
```

Comment:
`MCA` means "Multi Client Account", we used it by mistake in the variable `mcaSelectionOptions`. It would be better to use the acronym `GMC` in your method name.